arangod/Replication2/StateMachines/Document/DocumentLogEntry.h

```diff
@@ -44,21 +44,28 @@ enum class OperationType {
   kTruncate,
   kCommit,
   kAbort,
-  kAbortAllOngoingTrx
+  kAbortAllOngoingTrx,
+  kIntermediateCommit,
 };
 
-auto to_string(OperationType) noexcept -> std::string_view;
+template<class Inspector>
+auto inspect(Inspector& f, OperationType& p) {
+  return f.enumeration(p).values(
+      OperationType::kInsert, "Insert",                          //
```

Comment:
On the follower side we handle everything by doing a replace. However, I would like to keep those in case we want to change that in the future (provided we can figure out how we can do this by replicating only the changes and not the complete document).

Reply:
Okay :+1: